src/v2/Apps/Fair/Components/FairOverview/FairTimer.tsx

```diff
@@ -3,20 +3,25 @@ import { Box, Text } from "@artsy/palette"
 import { Timer } from "v2/Components/Timer"
 import { createFragmentContainer, graphql } from "react-relay"
 import { FairTimer_fair } from "v2/__generated__/FairTimer_fair.graphql"
+import { useTimer } from "v2/Utils/Hooks/useTimer"
 
 interface FairTimerProps {
   fair: FairTimer_fair
 }
 
 export const FairTimer: React.FC<FairTimerProps> = ({ fair: { endAt } }) => {
```

Comment:
is there any cases when `endAt` is `null`?

Reply:
I don't believe so. We have some presence validations in Gravity [here](https://github.com/artsy/gravity/blob/master/app/models/domain/fair.rb#L135).